chain/src/main/scala/org/bitcoins/chain/models/BlockHeaderDAO.scala

```diff
@@ -87,6 +87,18 @@ sealed abstract class BlockHeaderDAO
     table.filter(_.height === height).result
   }
 
+  def getBetweenHeights(from: Long, to: Long): Future[Vector[BlockHeaderDb]] = {
```

Comment:
probably should add a scaladoc saying there is no ordering guarantee here and it does not account for multiple headers at the same height

Reply:
Is the `to` inclusive or exclusive? It should be stated in the scaladoc. We should probably follow the convention of scala collections. 

I.e. 
```scala
1.until(3)
```

```scala
1.to(3)

```